shell/platform/windows/public/flutter_windows.h

```diff
@@ -0,0 +1,168 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_PUBLIC_FLUTTER_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_PUBLIC_FLUTTER_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "flutter_export.h"
+#include "flutter_messenger.h"
+#include "flutter_plugin_registrar.h"
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+// Opaque reference to a Flutter window controller.
+typedef struct FlutterDesktopWindowControllerState*
+    FlutterDesktopWindowControllerRef;
+
+// Opaque reference to a Flutter window.
+typedef struct FlutterDesktopWindow* FlutterDesktopWindowRef;
+
+// Opaque reference to a Flutter engine instance.
+typedef struct FlutterDesktopEngineState* FlutterDesktopEngineRef;
+
+
+// TODO: remove once the embedder project has swiched to native windows
```

Comment:
Why does this need to be added now and removed later? API compatibility with the GLFW version is (very much) a non-goal.

Reply:
Ignore.